### PR TITLE
Add workflow for push to master

### PR DESCRIPTION
**Pull request by gmalkov**
Also pulled out some values into variables

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -68,9 +73,9 @@ jobs:
     - name: Update dev deployment to use latest image
       run: |
         cd config
-        ./kustomize edit set image gcr.io/terra-kernel-k8s/workspace-test:0.0.0=gcr.io/terra-kernel-k8s/workspace-test:${{ github.sha }}
+        ./kustomize edit set image gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:0.0.0=gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:${{ github.sha }}
     - name: Deploy service directly to Dev namepsace
       run: |
-        gcloud container clusters get-credentials terra-kernel-k8s \
-                  --zone us-central1-a --project terra-kernel-k8s
+        gcloud container clusters get-credentials ${GOOGLE_PROJECT} \
```
Comment (zloery) [inline .github/workflows/gradle.yml]: These params are swapped

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -54,10 +59,10 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
-    - name: Grant execute permission for gradlew
+    - name: Grant execute permission for gradle
```
Comment (zloery) [inline .github/workflows/gradle.yml]: Actually gradlew, the name just looks weird

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (zloery) [reply .github/workflows/gradle.yml]: Aw multiline comments didn't work. This should cover ${GOOGLE_PROJECT} and ${GKE_CLUSTER}

Comment (gmalkov) [reply .github/workflows/gradle.yml]: Good catch!

**Comment by zloery** [comment]
LGTM

**Review by zloery — APPROVED** [review]

**Pull request merged**